src/main/java/lihua/tasks/Deadline.java

```diff
@@ -0,0 +1,60 @@
+package lihua.tasks;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+import org.json.simple.JSONObject;
+
+/**
+ * Represents a lihua.tasks.Deadline item, which is a child of lihua.tasks.Task
+ */
+public class Deadline extends Task {
+    private final LocalDate by;
```

Comment:
Perhaps a more intuitive name for the localdate variable will be better? I feel that just stating "by" could be hard to read for some.

Reply:
like `deadlineDate`? 🤔

Reply:
Hmmm i also agree with ZhangAnli this point. and deadlineDate seems good enough.

Reply:
I agree with Anli and Eriksen regarding this